lib/inspector.js

```diff
@@ -38,7 +37,7 @@ class Session extends EventEmitter {
     if (this[connectionSymbol])
       throw new ERR_INSPECTOR_ALREADY_CONNECTED('The inspector session');
     const connection =
-      new Connection((message) => this[onMessageSymbol](message));
+        new Connection((message) => this[onMessageSymbol](message));
```

Comment:
Please lint the file in order to fix the indentation.

Reply:
Linting results for JS files can be seen by running `make lint-js` or (on Windows) `vcbuild lint-js`.